acceptance/localcluster/localcluster.go

```diff
@@ -242,7 +242,9 @@ func (l *Cluster) initCluster() {
 	maybePanic(c.Wait())
 }
 
-func (l *Cluster) startNode(i int) *Container {
+// StartNode creates and starts the container corresponding
+// to node 'i'. The container is saved in l.Nodes.
+func (l *Cluster) StartNode(i int) {
```

Comment:
Was there a reason for exporting this? My unsent PR adds some locking here that will conflict with this change.